server/models/RawCapture.js

```diff
@@ -65,12 +66,21 @@ const rawCaptureFromRequest = (
     field_username: planter_identifier,
     field_user_photo_url: planter_photo_url,
     attributes,
-    created_at: new Date(timestamp).toISOString(),
-    updated_at: new Date(timestamp).toISOString()
-});
-
+    created_at: new Date().toISOString(),
+    updated_at: new Date().toISOString(),
+    capture_taken_at: new Date(timestamp * 1000).toISOString(),
+  });
 
-const RawCaptureCreated = ({ id, lat, lon, field_user_id, field_username, attributes, created_at }) => Object.freeze({
+const RawCaptureCreated = ({
+  id,
+  lat,
+  lon,
+  field_user_id,
+  field_username,
+  attributes,

```

Comment:
I didn't specify it in the issue, but I think it would be good to include the capture_taken_at to be included in the `RawCaptureCreated` event.